LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012, Caktus Consulting Group, LLC
+Copyright (c) 2015, Caktus Consulting Group, LLC
```

Comment:
I notice that both [RapidSMS](https://github.com/rapidsms/rapidsms/blob/develop/LICENSE) and [Django](https://github.com/django/django/blob/master/LICENSE) don't even specify years - maybe we could go that route?